storage-account.tf

```diff
@@ -10,6 +10,29 @@ locals {
   // for each client service two containers are created: one named after the service
   // and another one, named {service_name}-rejected, for storing envelopes rejected by bulk-scan
   client_service_names = ["bulkscanauto", "bulkscan", "sscs", "divorce", "probate", "finrem", "cmc", "publiclaw"]
+
+  resourcegroup_name          = "${azurerm_resource_group.rg.name}"
+
+  valid_subnets =  "${var.env == "aat"
+  ?
+  [
+    data.azurerm_subnet.jenkins_subnet.id,
+    data.azurerm_subnet.jenkins_aks_00.id,
+    data.azurerm_subnet.jenkins_aks_01.id,
+    data.azurerm_subnet.app_aks_00_subnet.id,
+    data.azurerm_subnet.app_aks_01_subnet.id,
+    data.azurerm_subnet.preview_aks_00_subnet.id,
+    data.azurerm_subnet.preview_aks_01_subnet.id,
+  ]
+  :
+  [
```

Comment:
would be better to extract this to a list and then use concat to join them https://www.terraform.io/docs/configuration/functions/concat.html